src/components/animations/CheckmarkAnimation.tsx

```diff
@@ -73,6 +77,69 @@ export function CheckmarkAnimation() {
     };
   };
 
+  const pulsingCheckmarkAnimation = useDerivedValue(() =>
```

Comment:
@jkadamczyk This is my first time using this lib so I'm not sure. I think I based it on an example from the docs or another animation that already existed in the codebase. 

My understanding is that `useSharedValue` is supposed to be used for stuff that's linked to events, like clicking a button to modify the shared value and animating it. This animation works on a timed loop as soon as it renders and nothing modifies its value so I'm not sure how I'd be able to chain it together otherwise.

If there's a better or more performant way to do this then I'd be happy to learn.

Reply:
Makes sense! I was asking cause I don't know either 😄 

I think @terrysahaidak may be in the know and give some advice in terms of performance and derived value usage